### ISSUE TITLE
Update Rmd template to meet web accessibility standards

### ISSUE THREAD

**Issue by robjhyndman** [rjournal/rjtools#126]
Comment from @ajrgodfrey about the Rmd template:

> It leads to a nice reading experience, but the HTML does not actually fully meet WCAG with for example too many heading level 1 tags on pages (sections are `<h1>` and should be `<h2>`) so there is an opportunity to upgrade the template to the latest and best WCAG has to offer.

Ref: https://www.w3.org/WAI/WCAG22/Techniques/html/H69.html
